planck-cljs/src/planck/io.cljs

```diff
@@ -126,6 +126,25 @@
   ([parent & more]
    (File. (apply str parent more))))
 
+(defn create-converter [f]
+  (partial reduce #(update-in % [%2] f)))
+
+(defn fstat
+  "Returns a map containing the attributes of the item at a given pathfil"
+  [path]
+  (let [as-int (create-converter js/parseInt)
```

Comment:
You can have the Obj-C side return numbers. No need to send strings over to be parsed.